package.json

```diff
@@ -12,7 +12,7 @@
 		"@wordpress/env": "9.2.0"
 	},
 	"scripts": {
-		"setup:tools": "yarn && composer install && TEXTDOMAIN=wporg composer exec update-configs && composer --working-dir=./source/wp-content/plugins/phpdoc-parser install",
```

Comment:
Not critic, but you would be running this command twice
`TEXTDOMAIN=wporg composer exec update-configs`

Reply:
Thanks Carlos! 
I'll remove this part from the PR